Cargo.toml

```
@@ -28,7 +28,7 @@ serde = { version = "1", default-features = false, features = ["alloc"], optiona
 serde_derive = { version = "1", default-features = false, optional = true }
 thiserror = { version = "1", optional = true }
 merlin = { version = "3", default-features = false }
-clear_on_drop = { version = "0.2", default-features = false }
+zeroize = { version = "1.1.0", default-features = false, features = ["alloc"], optional = true }
```

Other crates have updated to 1.5.  Lets try using that too.  I don't see any breaking changes between 1.1 and 1.5.
```suggestion
zeroize = { version = "1.5", default-features = false, features = ["alloc"], optional = true }
```